scapy/layers/inet.py

```diff
@@ -42,7 +43,7 @@ def whois(self):
         if WINDOWS:
             print whois(self.src)
         else:
-            os.system("whois %s" % self.src)
+            subprocess.call("whois %s" % self.src, shell=True, executable='/bin/bash')
```

Comment:
I'll remove os.
Subprocess uses by default `/bin/sh`, which do not provide `whois` (`os.system` used `/bin/sh` too). By specifying `shell=True`, (which allow python to call directly external programms like `whois`, without the path) you're allowed to specify a shell: here /bin/bash